R/s3store.r

```diff
@@ -9,10 +9,15 @@
 #' s3store(c(1,2,3), 'test123')
 #' print(s3read('test123'))
 #' # [1] 1 2 3
-#' }#' 
-s3store <- function(obj, name = NULL, .path = s3path(), ...) {
+#' }#'
+s3store <- function(obj, name = NULL, .path = s3path(), safe = TRUE, ...) {
   if (is.null(name)) name <- deparse(substitute(obj))
   s3key <- paste(.path, name, sep = '')
+  if (safe && s3mpi::s3exists(name, .path = s3path(), ...)) {
+    # using cat prints to stdout as opposed to messages, so it can be seen from syberia::run_model()
+    cat("An object with this name already exists. Use `safe = FALSE` to overwrite\n")
+    stop("------------------^")
```

Comment:
Also write out the actual name of the key in the message -- what if this is from deep in a call stack? The user won't know what is happening.